client/UI/FFRKViewInventory.cs

```diff
@@ -116,6 +118,34 @@ private void FFRKViewInventory_Load(object sender, EventArgs e)
             }
         }
 
+        private void FFRKViewInventory_Recalc()
+        {
+            if (DesignMode)
+                return;
+
+            if (FFRKProxy.Instance != null)
+            {
+                mAnalyzerSettings = AnalyzerSettings.DefaultSettings;
+                mAnalyzerSettings.LevelConsideration = TranslateLevelConsideration((ScoreUpgradeModeComboIndex)comboBoxScoreSelection.SelectedIndex);
+                mAnalyzer = new EquipmentAnalyzer(mAnalyzerSettings);
+
+                FFRKProxy.Instance.OnPartyList += FFRKProxy_OnPartyList;

```

Comment:
Something doesn't look right about this function.  It's being called from SelectedIndexChanged, but it's doing a bunch of initialization like setting up event handlers, creating a new mAnalyzerInstance, etc that looks like it should be one-time initialization such as that in an FFRKInventory_Load() function.  So probably something is wrong here.